### PR TITLE
Add initial project requirements 📎

### PR DESCRIPTION
**Pull request by vdemeester**
Add contributing and development docs, code of conduct and also github
templates.
Also adds sthaha as OWNER as he's gonna init the code from his initial
work 👼

Signed-off-by: Vincent Demeester <vdemeest@redhat.com>

### CODE REVIEW

**Comment by vdemeester** [comment]
cc @sthaha

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Contributing to Tekton Pipelines
```
Comment (chmouel) [inline CONTRIBUTING.md]: Do you want to mention the CLI here?

Comment (vdemeester) [reply CONTRIBUTING.md]: =)

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+   variable (e.g. `PROJECT_ID`).
+1. [Create a GKE cluster](https://github.com/knative/docs/blob/master/docs/install/Knative-with-GKE.md#creating-a-kubernetes-cluster)
+
+Note that
+[the `--scopes` argument to `gcloud container cluster create`](https://cloud.google.com/sdk/gcloud/reference/container/clusters/create#--scopes)
+controls what GCP resources the cluster's default service account has access to;
+for example to give the default service account full access to your GCR
+registry, you can add `storage-full` to your `--scopes` arg.
+
+## Environment Setup
```
Comment (sthaha) [inline DEVELOPMENT.md]: do you also want to add some notes on building

### Building Tekton Client

**Dependencies:**

[go mod](https://github.com/golang/go/wiki/Modules#quick-start) is used and required for dependencies.

**Building:**

```sh
$ go build ./cmd/...
```

It builds `tkn` binary in your current directory. You can start playing with it.

**Notes:**

- For building, Go `1.12` is required
- If you are building in your `$GOPATH` folder, you need to specify `GO111MODULE` for building it

```sh
# if you are building in your $GOPATH
GO111MODULE=on go build ./cmd/...
```

You can now try updating code for client and test out the changes by building the `tkn` binary.

Comment (vdemeester) [reply DEVELOPMENT.md]: done

**Comment by bobcatfish** [comment]
Aaaamazing, thanks so much for taking care of this @vdemeester !! ❤️ 🙏

**Pull request merged**